.github/workflows/ruby.yml

```diff
@@ -2,7 +2,7 @@ name: Ruby
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, feature/#33 ]
```

Comment:
This is only for my fork, so before we will merge this PR, I remove this branch filter.